### PR TITLE
fix sample indentation

### PR DESCRIPTION
**Pull request by sbodoff**

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -192,31 +195,33 @@ early and provides a statement URL that can be polled until it is complete:
...
-    'code': textwrap.dedent("""\
-    val NUM_SAMPLES = 100000;
-    val count = sc.parallelize(1 to NUM_SAMPLES).map { i =>
-    val x = Math.random();
-    val y = Math.random();
-    if (x*x + y*y < 1) 1 else 0
-    }.reduce(_ + _);
-    println(\"Pi is roughly \" + 4.0 * count / NUM_SAMPLES)
-    """)
+      'code': textwrap.dedent("""\
```
Comment (vanzin) [inline README.rst]: Need to fix indentation here:

```
'code': textwrap.dedent("""\
  val NUM_SAMPLES = 100000;
  val count = sc.parallelize(1 to NUM_SAMPLES).map { i =>
    val x = Math.random();
    val y = Math.random();
    if (x*x + y*y < 1) 1 else 0
  }.reduce(_ + _);
  println(\"Pi is roughly \" + 4.0 * count / NUM_SAMPLES)
""")
```

File: README.rst [hunk 2]
```diff
@@ -245,28 +251,34 @@ PySpark has the same API, just with a different initial request:
...
-    import random
-    NUM_SAMPLES = 100000
-    def sample(p):
-      x, y = random.random(), random.random()
-      return 1 if x*x + y*y < 1 else 0
-    count = sc.parallelize(xrange(0, NUM_SAMPLES)).map(sample).reduce(lambda a, b: a + b)
-    print "Pi is roughly %f" % (4.0 * count / NUM_SAMPLES)""")}
+    data = {
+      'code': textwrap.dedent("""
+      import random
```
Comment (vanzin) [inline README.rst]: I'd indent the embedded code one extra level. Same later in the R example.

**Comment by sbodoff** [comment]
I think I have fixed all the indent issues.

**Comment by vanzin** [comment]
Great, merging.

**Pull request closed without merging**